ui/articles-aem-renderer/src/main/res/layout/fragment_aem_article_item.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
```

Comment:
`ConstraintLayout` is not necessary

Reply:
the `ConstraintLayout` contains a single view that is set to take up all available space. This can be accomplished by making the `WebView` `match_parent` for the width and height and eliminate the extraneous ConstraintLayout